cockatrice/src/window_main.cpp

```diff
@@ -396,44 +401,44 @@ void MainWindow::loginError(Response::ResponseCode r, QString reasonStr, quint32
             break;
         }
         default:
-            QMessageBox::critical(this, tr("Error"), tr("Unknown login error: %1").arg(static_cast<int>(r)) + tr("\nThis usually means that your client version is out of date, and the server sent a reply your client doesn't understand."));
+            QMessageBox::critical(this, tr("Error"), tr("Unknown login error:") + QString(" %1<br>").arg(static_cast<int>(r)) + tr("This usually means that your client version is out of date, and the server sent a reply your client doesn't understand."));
```

Comment:
+1 for moving the `\n` out of the translatable string, but using `<br>` instead can be problematic (see previous comment)